src/styles/isomer-cms/elements/card.scss

```diff
@@ -194,9 +194,19 @@
       font-size: 18px;
       width: 100%;
       text-align: left;
+      line-height: 3.75rem;
     }
   }
 
+  &.siteNotificationSection {
+    h2 {
+      font-size: 1.25rem;
+      line-height: 2rem;
+      margin-bottom: 5px;
+    }
+    
+  }
```

Comment:
this style is effectively limited to `h2` that are below `.siteNotificationSection` - my worry here is if we add code here in the future and `siteNotificationSection` ends up becoming a wrapper, it could negatively affect nested h2s.

Should we instead scope it so that this is a single classname?

Reply:
That's a good point, modified in https://github.com/isomerpages/isomercms-frontend/pull/895/commits/1b98237e8c3907bf6661b07cd1c50a42e53ec781